src/Schema/Events/Toggle.ts

```diff
@@ -35,3 +35,31 @@ export interface ToggledNotification {
   original: number
   subject: PushNotificationType
 }
+
+/**
+ * A user toggles a saved search on/off on iOS
+ *
+ *  This schema describes events sent to Segment from [[toggledNotification]]
+ *
+ *  @example
+ *  ```
+ *  {
+ *    action: "toggledSavedSearch",
+ *    context_screen_owner_type: "artist",
+ *    context_screen_owner_id: "58de681f275b2464fcdde097",
+ *    context_screen_owner_slug: "anthony-hunter",
+ *    modified: 1,
+ *    original: 0,
+ *    saved_search_id: "58de681f275b2464fcdde097"
+ *  }
+ * ```
+ */
+export interface ToggledSavedSearch {
+  action: ActionType.toggledSavedSearch
+  context_screen_owner_type: ScreenOwnerType
+  context_screen_owner_id?: string
+  context_screen_owner_slug?: string
+  modified: number
+  original: number
```

Comment:
If we expect a boolean state, either "enabled" or "disabled", do our analytics support booleans or is the numeric 0/1 representation more compatible here?

Reply:
We can support booleans too. Do you suggest to update to "enabled": True/False, instead of modified/original?

Reply:
Sure, that would be great 👍🏼